app/models/miq_ae_datastore.rb

```diff
@@ -192,7 +190,7 @@ def self.datastores_for_reset
   end
 
   def self.default_domain_names
-    Vmdb::Plugins.system_automate_domains.collect(&:name)
+    domain_names.collect(&:name)
```

Comment:
I think this is supposed to be system domain (including those that are legacy), not all domains.